examples/dataset/semantic_textual_similarity/semantic_textual_similarity/constants.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2021-2024 Kolena Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+BUCKET = "kolena-public-examples"
```

Comment:
We should move the csv files to this bucket.

Reply:
Do I also have the non-raw version uploaded?

Reply:
Yes please!

Reply:
done